sharded_queue/storage.lua

```diff
@@ -1,50 +1,87 @@
-local utils = require('sharded_queue.utils')
-local queue_driver = require('sharded_queue.driver_fifottl')
+local log = require('log')
+local json = require('json')
+
+local DEFAULT_DRIVER = 'sharded_queue.drivers.fifottl'
+
+local queue_drivers = {}
+local function get_driver(driver_name)
+    if queue_drivers[driver_name] == nil then
+        queue_drivers[driver_name] = require(driver_name)
```

Comment:
May be a non-existent name, can be moving to validate_config?

Reply:
Sure, thx.